scripts/obtain/install.sh

```diff
@@ -4,8 +4,25 @@
 
 # Note: This script should be compatible with the dash shell used in Ubuntu. So avoid bashisms! See https://wiki.ubuntu.com/DashAsBinSh for more info
 
-#setup some colors to use. These need to work in fairly limited shells, like the Ubuntu Docker container where there are only 8 colors.
-#See if stdout is a terminal
+# Stop script on NZEC
```

Comment:
@ellismg what happens if bash isn't available on whatever distro?

Would `#!/usr/bin/env sh` be more portable?